.github/actions/artifacts/download-codecov-cli.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+
+curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
+curl -Os https://cli.codecov.io/latest/linux/codecov
```

Comment:
It would be better to see these commands in the log.

```suggestion

set -x
curl https://keybase.io/codecovsecurity/pgp_keys.asc |
    gpg --no-default-keyring --keyring trustedkeys.gpg --import
curl -Os https://cli.codecov.io/latest/linux/codecov
```